src/state/sagas/runner/UpdateRunner/UpdateRunner.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+import { t } from 'i18next';
+import { put, takeEvery, call, select } from 'redux-saga/effects';
+import { Api } from '../../../../services/config/Api';
+import { RUNNER_ACTIONS_KEY } from '../../../commons/RunnerConstants';
+import { dispatchSetApplicationErrorMessage } from '../../../dispatchers/app/ApplicationDispatcher';
+
+const getOrganizationId = (state) => state.organization.current.data.id;
+const getWorkspaceId = (state) => state.workspace.current.data.id;
+
+export function* updateRunner(action) {
+  const runnerId = action.runnerId;
+  try {
+    const organizationId = yield select(getOrganizationId);
+    const workspaceId = yield select(getWorkspaceId);
+    const newRunnerDetails = action.runner;
```

Comment:
I might be wrong, but isn't the action content looking like this:
```
{
  type: RUNNER_ACTIONS_KEY.TRIGGER_SAGA_UPDATE_RUNNER,
  runnerId,
  newRunnerDetails,
}
```
In the dispatcher, the action does not contain any `runner` property